sdk/communication/azure-communication-sms/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ Updated `azure-communication-sms` version
 - Added support for tagging SMS messages.
 - Send method series in SmsClient are idempotent under retry policy.
 - Added `SmsOptions`
+- Added implementation of the repeteabilityRequestId.
```

Comment:
No. The reality is this was not fixed in 03-39 release but the latest. Move this to be behind 1.1.0-beta.1